include/PR/gbi.h

```diff
@@ -16,6 +16,9 @@
 
 #define F3DEX_GBI_2
 
+/* F3DEX2 with Point Lighting */
+#define F3DEX_GBI_PL
```

Comment:
Actually I wonder if we should pass this define (and `F3DEX_GBI_2`) in the Makefile instead of just defining it here

Reply:
Would we want to auto define `F3DEX_GBI_2` if `F3DEX_GBI_PL` is passed, so we only had to pass `F3DEX_GBI_PL` on the makefile? Something like

```c
#ifdef F3DEX_GBI_PL
#define F3DEX_GBI_2
#endif
```

Reply:
I think this is a good idea and is how it's supposed to work, but I'd like to do this separately. As mentioned above, I have plans for a formatting pass on this file and it seems like a good time to do this then.